frontend/lab/samples/ifs/pi.md

```diff
@@ -124,7 +124,6 @@ And at the bottom of the loop, add:
 ```evy
 while // ...
   // ...
-  pi := reds / i * 4
```

Comment:
This is estimating π, isn't it? So you should not be removing this I think. Perhaps this sample could show the delta between the builtin π and the estimated π?

Reply:
that's a great idea!

Reply:
fixed.